components/esm/cellref.cpp

```diff
@@ -67,6 +67,10 @@ void ESM::CellRef::loadData(ESMReader &esm, bool &isDeleted)
                 break;
             case ESM::FourCC<'X','S','C','L'>::value:
                 esm.getHT(mScale);
+                if (mScale < 0.5)
+                    mScale = 0.5;
+                else if (mScale > 2)
+                    mScale = 2;
```

Comment:
Probably it would be better to cap scale during saving instead.

Reply:
It'd have to be in addition to deal with content files containing uncapped scales - of course those shouldn't exist, but...

Think it'd be worthwhile to put the capping into its own function so it can be used in both places?

Reply:
I honestly don't think clamping needs to be its own function, the code will be readable enough anyways. We can't remove clamping upon loading as old saves don't necessarily have the scale clamped. Clamping upon saving won't break anything but it might be redundant anyway.